crates/core/src/orphan.rs

```diff
@@ -4,16 +4,15 @@ use itertools::Itertools;
 use marzano_language::{language::Language, target_language::TargetLanguage};
 use marzano_util::cursor_wrapper::CursorWrapper;
 use marzano_util::position::Range;
-use tree_sitter::{Parser, Range as TSRange, Tree};
+use tree_sitter::{Parser, Tree};
 
-fn merge_treesitter_ranges(ranges: Vec<TSRange>) -> Vec<Range> {
+fn merge_treesitter_ranges(ranges: Vec<Range>) -> Vec<Range> {
```

Comment:
Since the goal is to make this TS-agnostic, rename?

```suggestion
fn merge_ranges(ranges: Vec<Range>) -> Vec<Range> {
```